packages/marshal/src/marshal.js

```diff
@@ -168,15 +168,25 @@ export const makeMarshal = (
      * a canonical-json stringify of the encoded form.
      *
      * @param {Passable} val
+     * @param {string} key
```

Comment:
Why is the `key` second? I'd expect either
```suggestion
     * @param {Passable} val
     * @param {string} [key]
```
or
```suggestion
     * @param {string} key
     * @param {Passable} val
```